code/src/App.js

```diff
@@ -1,9 +1,148 @@
-import React from 'react'
+import React, { useState } from "react"
+import "./index.css"
+
+const characters = ["Rudolph", "The Grinch", "Frosty the Snowman"]
+const foods = [
+  "gingerbread cookies",
+  "anything as long you can have glögg to it",
+  "just coffee"
+]
 
 export const App = () => {
+  const [name, setName] = useState()
+  const [nickname, setNickname] = useState()
+  const [character, setCharacter] = useState("Rudolph")
+  const [food, setFood] = useState("gingerbread cookies")
+  const [song, setSong] = useState("")
+  const [showResult, setShowResult] = useState(false)
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowResult(true)
+  }
+
   return (
     <div>
-      Find me in src/app.js!
+      <header>
+        <h1>How much do you love Christmas?</h1>
+      </header>
+
+      {!showResult && (
+        <form onSubmit={handleSubmit}>
+          <div className='survey'>
+            <label for='name'>Fill in your first name, please</label>
+            <input
+              id='name'
+              type='text'
+              onChange={event => setName(event.target.value)}
+              required
+              value={name}
+            />
+
+            <div className='divider'>
+              * * * * * * * * * * * * * * * * * * * * *
```

Comment:
Note that is someone has help reading the page (like a screen reader) it will way "*" here. So maybe it's better to add this to the CSS instead - then it will not read it